src/components/TodoContentFooter.tsx

```diff
@@ -0,0 +1,77 @@
+import { FC } from 'react';
+import classNames from 'classnames';
+import { getEnumKeys } from '../helpers/getEnumKeys';
+import { TodoStatus } from '../types/TodoStatus';
+import { useTodoContext } from '../context/todoContext/useTodoContext';
+import { todosApi } from '../api/todos-api';
+import { useErrorContext } from '../context/errorContext/useErrorContext';
+
+interface TodoContentFooterProps {
+  selectedStatusTodo: TodoStatus,
+  onSelectStatusTodo: (status: TodoStatus) => void
+}
+
+export const TodoContentFooter: FC<TodoContentFooterProps> = (props) => {
+  const {
+    selectedStatusTodo,
+    onSelectStatusTodo,
+  } = props;
+
+  const { todos, setHandlingTodoIds } = useTodoContext();
```

Comment:
The `useTodoContext` hook is called twice. Instead of calling it twice, call it only once and destructure all necessary variables at once.